REPOSITORY-GUIDELINES.md

```diff
@@ -0,0 +1,96 @@
+# Repository Guidelines
+
+This document describes the scope and lifecycle of repositories within [Cilium](https://github.com/cilium).
+
+You can find the list of all repositories, sorted by their scope, in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md).
+
+**Table of Contents**
+
+* [Scope](#scope)
+  * [```Core```](#core)
+  * [```Ecosystem```](#ecosystem)
+  * [```Infra```](#infra)
+  * [```Special```](#special)
+  * [```Historical```](#historical)
+* [Lifecycle](#lifecycle)
+  * [Voting](#voting)
+  * [Addition](#addition)
+  * [Change of scope](#change-of-scope)
+* [Credit](#credit)
+
+## Scope
+
+Each repository within the Cilium GitHub organization is assigned one of five scopes: [```Core```](#core), [```Ecosystem```](#ecosystem), [```Infra```](#infra), [```Special```](#special), or [```Historical```](#historical). This assigned scope helps to convey each repository's purpose and responsibility in the overall functioning of the project.
+
+### ```Core```
+
+Core repositories are at the heart of the Cilium project, and represent its most essential functioning. They are the most active in terms of both contributions and use cases, and represent the repositories that adopters will typically interact with most often.
+
+Notable repositories of this type include [Cilium](https://github.com/cilium/cilium), [eBPF-go](https://github.com/cilium/ebpf), [Hubble](https://github.com/cilium/hubble), and [Tetragon](https://github.com/cilium/tetragon).
+
+You can find the list of all ```Core``` repositories in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md#core-scope).
+
+### ```Ecosystem```
+
+Ecosystem repositories are extensions of the core project, providing optional functionality that adopters may find useful. While not considered essential, they play a pivotal role of addressing a range of use cases and needs for Cilium adoptors.
+
+Examples of repositories of this type include useful Go libraries like [Fake](https://github.com/cilium/fake), or repositories designed to integrate Cilium with other projects like [Proxy](https://github.com/cilium/proxy).
+
+You can find the list of all ```Ecosystem``` repositories in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md#ecosystem-scope).
+
+### ```Infra```
+
+Infrastructure repositories are those that are required to keep the Cilium project running smoothly, and are not designed to be used by adopters.
+
+Examples of repositories of this type include development and testing tools like [little-vm-helper](https://github.com/cilium/little-vm-helper), or those helping with the overall contribution process like [github-actions](https://github.com/cilium/github-actions).
+
+You can find the list of all ```Infra``` repositories in [REPOSITORIES.md](https://github.com/cilium/community/blob/main/REPOSITORIES.md#infra-scope).
+
+### ```Special```
```

Comment:
I wonder whether we need to differentiate between "Infra" and "Special". Both seem to be repositories we use as a project to coordinate and develop functionality in the Core / Ecosystem categories.

Why keep them separate? Or, what would a combined category look like? For instance, would we name it `Internal` or `Supporting repositories` or something like that?

Reply:
That is true! I like combining them, and I like the name ```Internal```.

At one point we also had statuses, and so it didn't make sense to give something like cilium/community or forks a status. That is why they were originally separated into two groups.